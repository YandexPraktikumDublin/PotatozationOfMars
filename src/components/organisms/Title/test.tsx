import React from 'react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { Title } from '.'

describe('<Title />', () => {
  it('should renders correct <Title />', () => {
    const wrapper = shallow(<Title />)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
