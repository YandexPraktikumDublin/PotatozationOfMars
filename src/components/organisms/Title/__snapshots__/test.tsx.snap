// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Title /> should renders correct <Title /> 1`] = `
<h1
  className="text-4xl mb-4 dark:text-white"
>
  test
</h1>
`;
